index.js

```diff
@@ -40,6 +40,25 @@
       };
     };
 
+    factory.assocMany = function(numElements, name, key, attrs) {
```

Comment:
This method should take args `function(name, key, num, attrsArray)` for consistency with the other methods. Sorry, I should have picked this up before.